docs/installation/oracle.md

```diff
@@ -100,10 +157,10 @@ on how to create and mount btrfs filesystems.
 To enable btrfs support on Oracle Linux:
 
 1. Ensure that `/var/lib/docker` is on a btrfs filesystem.
-1. Edit `/etc/sysconfig/docker` and add `-s btrfs` to the `OTHER_ARGS` field.
-2. Restart the Docker daemon:
 
-You can now continue with the [Docker User Guide](/userguide/).
+2. Edit `/etc/sysconfig/docker` and add `-s btrfs` to the `OTHER_ARGS` field.
```

Comment:
We may need to check if this still works on Oracle 7. I think we don't use this file for SystemD based systems